README.md

```diff
@@ -255,8 +255,97 @@ created in Step3 is listed
 
 
 ### 4. Develop Watson Knowledge Studio model
-Detailed steps to create a model in WKS is provided [here](https://github.ibm.com/IBMCode/howtos/tree/master/extracting-personal-data-from-unstructured-text-using-watson-knowledge-studio). 
-This is a how-to guide. Follow these steps to create a custom model.
+## 4.1 Import Artifacts
+In github repository, navigate to WKS folder. Download the files named “Documents.zip” and
```

Comment:
This should provide info on which GitHub repository your are referring to.
i.e. for gdpr-fingerprint-pii, you are asking to go to gdpr-fingerprint-pii/WKS,
but since this is a how to, you should also make the instructions generic and explain what these files are, so the instructions aren't just specific to one journey.

Reply:
I have added detailed instructions and provided reference links in case user wants to know details of Type Systems and Documentation. Regarding the repository url - Since the instructions here will be moved to how-to repository, for now I am referring to repositories as just github repository and not providing the url. Hope that's fine.